### PR TITLE
[VEN-1895] Add Arbitrum sequencer downtime validation for Chainlink Oracle

### PR DESCRIPTION
**Pull request by 0xlucian**
## Description

This PR adds Chainlink Oracle support for Arbitrum.

Resolves #VEN-1895

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
> @0xlucian Instead of creating a separate contract I think we can add a `sequencerUptimeFeed` immutable address. If it's not zero then verify the sequencer uptime. I think this way we don't need to maintain multiple chainlink oracle contracts.
> 
> Also sequencer is used in OP, ARB and Base networks. So if we decide to use a separate contract, then we should rename the contract from `ArbiChainlinkOracle ` to `SequencerChainlinkOracle`

I would rename the contract to `SequencerChainlinkOracle`. I would personally keep it in its own contract, to avoid deploying code to networks that we won't use it

**Review by chechu — APPROVED** [review]

**Review by Debugger022 — APPROVED** [review]

**Review by GitGuru7 — APPROVED** [review]

**Review by kkirka — APPROVED** [review]
LGTM overall

File: contracts/oracles/SequencerChainlinkOracle.sol [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/**
+    @title Sequencer Chain Link Oracle
+    @notice Oracle to fetch price using chainlink oracles on L2s with sequencer
+*/
+contract SequencerChainlinkOracle is ChainlinkOracle {
+    /// @notice L2 Sequencer feed
+    AggregatorV3Interface public immutable sequencer;
+
+    /// @notice L2 Sequencer grace period
+    uint256 public constant GRACE_PERIOD_TIME = 3600;
```
[the remark this answers is not in the document]
Comment (kkirka) [reply contracts/oracles/SequencerChainlinkOracle.sol:17]: Alternatively, we could create a separate section for the oracle configuration in the docs, where we could list all these parameters. This way risk teams (Chaos Labs, Findeci, Gauntlet, etc.) could assess those and recommend risk params taking the potential price lags into account.

[the remark this answers is not in the document]
Comment (chechu) [reply contracts/oracles/SequencerChainlinkOracle.sol:17]: aave uses 3,600 for their [PriceOracleSentinel contract](https://arbiscan.io/address/0x7A9ff54A6eE4a21223036890bB8c4ea2D62c686b) on Arbitrum (and Optimism). We can start with this value

[the remark this answers is not in the document]
Comment (kkirka) [reply contracts/oracles/SequencerChainlinkOracle.sol:17]: I suggest we still notify the risk teams about this

**Pull request merged**